src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -105,4 +115,19 @@ const StyledIcon = styled(Icon)`
   }
 `;
 
+const Notice = styled.div`
+  margin-left: 184px;
+  display: flex;
+`;
```

Comment:
This can be the Flex component. Just use the `otherProperties` prop to pass marginLeft.